lib/capistrano/tasks/wordpress.rake

```diff
@@ -0,0 +1,35 @@
+namespace :wordpress do
+ 
+  desc "Install WP cli"
+  task :install_executable do
+    on roles(:web) do
+      within shared_path do
```

Comment:
Any idea why it doesn't install the binary in the shared_path (but in the home directory) ?

Reply:
the shared dire is created? did you try to print the `shared_path` value?